src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1162,6 +1168,18 @@ impl GraphEditorModel {
     fn scene(&self) -> &Scene {
         self.app.display.scene()
     }
+
+    fn set_style(&self, is_light:bool) {
+        if is_light { self.app.themes.set_enabled(&["dark"])  }
+        else        { self.app.themes.set_enabled(&["light"]) }
+    }
+
+    fn is_style_light(&self) -> bool {
+        let styles     = StyleWatch::new(&self.app.display.scene().style_sheet);
+        let fallback   = color::Lcha::new(0.0,0.0,0.0,0.7);
+        let text_color = styles.get_color_or("application.text.color", fallback);
+        text_color.lightness <= 0.5
+    }
```

Comment:
this is a hack. Remove it! You have created a bool `style_light` for that, right? Moreover, you dont need this variable at all, as you are remembering the current state (light / dark) in the frp network.